content/Development/qtest.md

```diff
@@ -0,0 +1,224 @@
+---
+title: "QFile Query Unit Test"
```

Comment:
I created a dedicated entry because I often need to explain how to set up qtest, which requires pretty long content.
But if we didn't want to have a dedicated page, I will move it to either of DeveloperGuid, Hive Developer FAQ, or How to Contribute.

Reply:
i wouldn't call it unit, maybe functional tests. Otherwise, people start abuse and instead of simple unit test and a new q file